src/core/src/op/tile.cpp

```diff
@@ -88,16 +62,13 @@ bool op::v0::Tile::evaluate_tile(const HostTensorVector& outputs, const HostTens
     ov::Shape data_shape = data->get_shape();
     auto data_rank = data_shape.size();
     auto output_rank = std::max(data_rank, repeats_rank);
-
-    // expand data shape and repeats to output rank
-    data_shape.insert(data_shape.begin(), output_rank - data_rank, 1);
     repeats_val.insert(repeats_val.begin(), output_rank - repeats_rank, 1);
```

Comment:
Please remove variables that don't use after shape_infer usage

Reply:
`repeats_val` must be expanded with size of `output_rank`  before `ngraph::runtime::reference::tile` because the `ngraph::runtime::reference::tile` assumes the `repeats_val` has the size of output_rank, otherwise it has index overflow. Those variables are kept to fill the correct shape of `repeats_val`.  Let me try to expand `repeats_val` according to the shape of the `output_shape ` Thanks

Reply:
Now I see, thanks for the explanation

Reply:
Great Suggestions, I remove useless variables in 05f5e38098c6497bed4b2b1b239bc264a3540420